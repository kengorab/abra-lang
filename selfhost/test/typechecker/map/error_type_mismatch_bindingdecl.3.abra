val m: Map<Int, String> = { (123): Option.None_, (456): Option.Some("hello") }