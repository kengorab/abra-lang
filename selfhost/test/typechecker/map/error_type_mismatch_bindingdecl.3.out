Error at %FILE_NAME%:1:42
Type mismatch
  |  val m: Map<Int, String> = { (123): Option.None_, (456): Option.Some("hello") }
                                              ^
Expected: String
but instead found: <unknown>?
